src/misc_gui.cpp

```diff
@@ -118,17 +118,17 @@ class LandInfoWindow : public Window {
 #else
 #	define LANDINFOD_LEVEL 1
 #endif
-		Debug(misc, LANDINFOD_LEVEL, "TILE: 0x{:x} ({},{})", (TileIndex)tile, TileX(tile), TileY(tile));
-		Debug(misc, LANDINFOD_LEVEL, "type   = 0x{:x}", tile.type());
-		Debug(misc, LANDINFOD_LEVEL, "height = 0x{:x}", tile.height());
-		Debug(misc, LANDINFOD_LEVEL, "m1     = 0x{:x}", tile.m1());
-		Debug(misc, LANDINFOD_LEVEL, "m2     = 0x{:x}", tile.m2());
-		Debug(misc, LANDINFOD_LEVEL, "m3     = 0x{:x}", tile.m3());
-		Debug(misc, LANDINFOD_LEVEL, "m4     = 0x{:x}", tile.m4());
-		Debug(misc, LANDINFOD_LEVEL, "m5     = 0x{:x}", tile.m5());
-		Debug(misc, LANDINFOD_LEVEL, "m6     = 0x{:x}", tile.m6());
-		Debug(misc, LANDINFOD_LEVEL, "m7     = 0x{:x}", tile.m7());
-		Debug(misc, LANDINFOD_LEVEL, "m8     = 0x{:x}", tile.m8());
+		Debug(misc, LANDINFOD_LEVEL, "TILE: {:#x} ({},{})", (TileIndex)tile, TileX(tile), TileY(tile));
```

Comment:
This the fmt way to achieve 0x... formatting, search for "binary" on this page https://hackingcpp.com/cpp/libs/fmt.html

Reply:
I know, but did you know that `{:#04x}` with 0 yields `0x00`? So you're not getting 4 digits, you're getting 4 characters.
That's why I reverted to `0x{:x}`, even though I know there's a flag adding the `0x`. Just because with the flag it will mess with you when setting the number of digits.